drivers/overlay/overlay.go

```diff
@@ -2546,6 +2546,20 @@ func nameWithSuffix(name string, number int) string {
 	return fmt.Sprintf("%s%d", name, number)
 }
 
+func checkIfAllNesseseryFilesExistsInLayerStore(target string) error {
+	for _, p := range []string{
+		filepath.Join(target, "diff"),
+		filepath.Join(target, "info"),
+		filepath.Join(target, "blob"),
+	} {
+		if err := fileutils.Exists(p); err != nil {
+			wrapped := fmt.Errorf("failed to stat additional layer %q: %w", p, err)
+			return fmt.Errorf("%v: %w", wrapped, graphdriver.ErrLayerUnknown)
```

Comment:
_If_ there will be a loop and we don’t decide to just support one value:

This error now exists only to be (not?) logged in the loop; the ultimate `ErrLayerUnknown` error returned to callers of `getAdditionalLayerPath` is created elsewhere.

So I think we can avoid the `ErrLayerUnknown` use, and just format an error string that makes sense in the logs.